File: 1791-richest-customer-wealth/richest-customer-wealth.java
class Solution {
    public int maximumWealth(int[][] accounts) {
        int m = accounts.length;
        int n=accounts[0].length;
        int w,maxwel=0;
        for(int i=0;i<m;i++){
            w=0;
            for(int j=0;j<n;j++){
                w+=accounts[i][j];
            }
            maxwel=Math.max(maxwel,w);
        }
        return maxwel;
    }
}

File: 1791-richest-customer-wealth/richest-customer-wealth.py
class Solution:
    def maximumWealth(self, accounts: list[list[int]]) -> int:
        m = len(accounts)
        b =0
        for i in range(m):
            n = len(accounts[i])
            sum = 0
            for j in range(n):
                sum += accounts[i][j]
            if(sum > b):
                b =sum
        return b